### PR TITLE
Junos security monitoring info

### PR DESCRIPTION
**Pull request by eesperi**
Added a collector to gather metrics from the JunOS command 'security show monitoring'.

### CODE REVIEW

File: security/collector.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	memoryUtilization  *prometheus.Desc
+	currentFlowSession *prometheus.Desc
+	maxFlowSession     *prometheus.Desc
+	currentCpSession   *prometheus.Desc
+	maxCpSession       *prometheus.Desc
+)
+
+func init() {
+	l := []string{"target", "re_name"}
+	//l := []string{"target", "re_name", "description", "name"}
```
Comment (czerwonk) [inline security/collector.go:27]: please remove comments when not needed any more

**Review by czerwonk — CHANGES_REQUESTED** [review]
Happy to merge when comment is removed

File: security/collector.go (hunk 1 above)
Comment (eesperi) [reply security/collector.go]: I've removed the comment.

**Comment by czerwonk** [comment]
Thanks for your contribution 🙂

**Pull request merged**